### DISCUSSION TITLE
Competitive Rank

### DISCUSSION THREAD

**Discussion by AKaravas** [markus-wa/demoinfocs-golang#292] [Q&A] [answered]
Hello. I am using the https://github.com/pnxenopoulos/csgo package to parse the demo. The package is using this repository in order to get the demo parsed. This exports a very big json file. Is there a place where i can read the rank of the players? I search for keywords like "rank" or "Gold Nova" or "GN" but i didnt find anything. Is it possible to get it from the demo file? 

Best regards

**Comment by markus-wa** [comment, marked as answer]
:wave: @AKaravas - unfortunately I'm unfamiliar with the format exported by `pnxenopoulos/csgo`.

But when using the Go library, the rank info should be available in the `RankUpdate` event for MM demos: https://pkg.go.dev/github.com/markus-wa/demoinfocs-golang/v2@v2.8.1/pkg/demoinfocs/events#RankUpdate

You can check the readme for how to capture events.

Hope that helps :slightly_smiling_face:

**Reply by AKaravas** [reply]
awesome! Thank you very much :)
